app/domain/logs.rb

```diff
@@ -590,6 +590,20 @@ module AuthnJwt
         code: "CONJ00116D"
       )
 
+      MandatoryClaimsDenyListValue = ::Util::TrackableLogMessageClass.new(
+        msg: "Mandatory claims deny list value is '{0-deny-claims-list}'",
+        code: "CONJ00117D"
+      )
+
+      ValidatingClaimName = ::Util::TrackableLogMessageClass.new(
+        msg: "Validating claim name '{0-claim-name}'",
+        code: "CONJ00118D"
+      )
+
+      ValidatedClaimName = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully validated claim name '{0-claim-name}'",
+        code: "CONJ00119D"
+      )
```

Comment:
same comment -
1. I'm not sure we the phrase 'claim name' is correct, I think it should be either claim or claim key.
2. denylist in one word -> see [this](https://ca-il-confluence.il.cyber-ark.com/display/rndp/%5BSpike%5D+Rename+and+offensive+of+conjur#id-[Spike]Renameandoffensiveofconjur-Specificchanges)

Reply:
@shulifink WDYT?

Reply:
Done